simulator-core/src/main/java/com/evbox/everon/ocpp/simulator/station/handlers/ocpp/support/AvailabilityStateMapper.java

```diff
@@ -0,0 +1,33 @@
+package com.evbox.everon.ocpp.simulator.station.handlers.ocpp.support;
+
+import com.evbox.everon.ocpp.simulator.station.evse.EvseState;
+import com.evbox.everon.ocpp.v20.message.station.ChangeAvailabilityRequest.OperationalStatus;
+import com.google.common.collect.ImmutableMap;
+
+import java.util.Map;
+
+import static com.evbox.everon.ocpp.simulator.station.evse.EvseState.AVAILABLE;
+import static com.evbox.everon.ocpp.simulator.station.evse.EvseState.UNAVAILABLE;
+import static com.evbox.everon.ocpp.v20.message.station.ChangeAvailabilityRequest.OperationalStatus.INOPERATIVE;
+import static com.evbox.everon.ocpp.v20.message.station.ChangeAvailabilityRequest.OperationalStatus.OPERATIVE;
+
+/**
+ * Mapper from request status to EVSE state.
+ */
+public class AvailabilityStateMapper {
+
+    private static final Map<OperationalStatus, EvseState> MAPPER = ImmutableMap.of(
+            OPERATIVE, AVAILABLE,
```

Comment:
is it possible to align literals with spec and remove this mapper?

Reply:
The spec says: _When changing the availability of an EVSE to Operative, the status of the EVSE has changed to Available, Occupied or Reserved.When changing the availability of an EVSE to Inoperative , the status of the EVSE has changed toUnavailable or Faulted._

To remove it you need to use OperationalStatus in EVSE and this is not what spec says